letsencrypt-auto-source/pieces/bootstrappers/alpine_common.sh

```diff
@@ -0,0 +1,31 @@
+BootstrapAlpineCommon() {
+  # Tested with:
+  #   - Alpine 3.4
+
+  $SUDO apk update || echo apk update hit problems but continuing anyway...
```

Comment:
Not sure why you have to repeat this, I guess it must be a bad design decision from the project itself. In any case, same comment as above.